bundle/default-bundle/Dockerfile-focal

```diff
@@ -1,6 +1,9 @@
 FROM eclipse-temurin:17.0.8_7-jre-focal
 
-RUN mkdir /opt/app
+# The /opt/app is used for the Connectors runtime, and out-of-the-box connectors
+# Use the /opt/additionalLibs to mount your own connectors, secret providers, or include other jars into the classpath
+RUN mkdir /opt/app && mkdir /opt/additionalLibs
```

Comment:
What if we use something simpler like `/opt/custom`? This would avoid user problems as snakeCaseOftenLeadsToProblems 😄

Reply:
`extras` works for me as well. Thought about custom because we call it custom connector and custom secret provider.